### PR TITLE
Add style guide byline cleaners

### PR DESCRIPTION
**Pull request by currysoup**
Often our photographers will submit their names in a way that doesn't conform to our style guide. 

This adds a single cleaner that applies three of the major causes of style guide violations in photographer bylines. Curly quotes rather than straight quotes, removing the full stop after an initial and squishing together pairs of initials.

### CODE REVIEW

**Comment by paperboyo** [comment]
Amazlang! For the record, see: [initials](https://www.theguardian.com/guardian-observer-style-guide-i) and [quotation marks](https://www.theguardian.com/guardian-observer-style-guide-q).

**Comment by currysoup** [comment]
Kicked the builds - seem to be OK now!

File: common-lib/src/test/scala/com/gu/mediaservice/lib/cleanup/GuardianStyleBylineTest.scala [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    cleanedMetadata.byline should be (Some("First AB"))
+  }
+
+  it("should remove dots in initials and insert spaces in unusual cases") {
+    val metadata = createImageMetadata("byline" -> "Ishara S.kodikara")
+    val cleanedMetadata = GuardianStyleByline.clean(metadata)
+
+    // NOTE: The capitalisation cleaner should handle this becoming Title Case
+    cleanedMetadata.byline should be (Some("Ishara S kodikara"))
```
Comment (akash1810) [inline common-lib/src/test/scala/com/gu/mediaservice/lib/cleanup/GuardianStyleBylineTest.scala]: is it worth having a test that chains these two tests together to be explicit?

Comment (currysoup) [reply common-lib/src/test/scala/com/gu/mediaservice/lib/cleanup/GuardianStyleBylineTest.scala]: Yeah I think this is something that could certainly be added to make sure we don't get any weird behaviour!

**Comment by paperboyo** [comment]
Nobody expected Spanish Inquisition:
`M.A.PUSHPA KUMARA` changed to `Ma Pushpa Kumara`
not that we _have to_ fix everybody’s _every_ mistake… I can add it to the dictionary, but it seems to be spelled all over the place, too.

**Comment by currysoup** [comment]
Ugh actually yeah we'll need to put the initial squishier after the capitaliser other wise it will turn `CP Scott` into `Cp Scott`.

**Comment by currysoup** [comment]
Will do that tomorrow - should take 10 seconds!

**Pull request merged**